distribution/lib/Standard/Table/0.0.0-dev/src/Data/Join_Kind_Cross.enso

```diff
@@ -0,0 +1,11 @@
+from Standard.Base import all
+
+type Join_Kind_Cross
+    ## Cartesian product: each row of the left table is paired with each row of
+       the right table.
+    Cross
+
+    ## PRIVATE
+       Returns the SQL representation of this join kind as text.
+    to_sql : Text
+    to_sql self = "INNER JOIN"
```

Comment:
Shouldn't this be `CROSS JOIN`?
I know `INNER JOIN` with no condition results in the same, but better to be explicit.

Reply:
Done